translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"pods'.\n"
+"\n"
+"\t\tPara fazer o proxy the todas as apis do kubernetes em um caminho "
+"diferente, utilize:\n"
+"\n"
+"\t\t    $ kubectl proxy —api-prefix=/custom/\n"
+"\n"
+"\t\tCom o comando acima você pode fazer 'curl localhost:8001/custom/api/v1/"
+"pods'"
+
+#: pkg/kubectl/cmd/patch.go:59
+msgid ""
+"\n"
+"\t\tUpdate field(s) of a resource using strategic merge patch\n"
+"\n"
+"\t\tJSON and YAML formats are accepted.\n"
+"\n"
+"\t\tPlease refer to the models in https://htmlpreview.github.io/?https://"
+"github.com/kubernetes/kubernetes/blob/HEAD/docs/api-reference/v1/definitions."
+"html to find if a field is mutable."
+msgstr ""
+"\n"
+"\t\tAtualizar o(s) campo(s) de um recurso usando strategic merge patch\n"
+"\n"
+"\t\tFormatos JSON e YAML são aceitos.\n"
+"\n"
+"\t\tConsulte os modelos em https://htmlpreview.github.io/?https://github.com/"
+"kubernetes/kubernetes/blob/HEAD/docs/api-reference/v1/definitions.html para "
+"descobrir se um campo é mutável."
+
+#: pkg/kubectl/cmd/label.go:70
+#, c-format
+msgid ""
+"\n"
+"\t\tUpdate the labels on a resource.\n"
+"\n"
+"\t\t* A label must begin with a letter or number, and may contain letters, "
+"numbers, hyphens, dots, and underscores, up to %[1]d characters.\n"
+"\t\t* If --overwrite is true, then existing labels can be overwritten, "
+"otherwise attempting to overwrite a label will result in an error.\n"
+"\t\t* If --resource-version is specified, then updates will use this "
+"resource version, otherwise the existing resource-version will be used."
+msgstr ""
+"\n"
+"\t\tAtualiza labels em um recurso.\n"
+"\n"
+"\t\t* Um label deve começar com uma letra ou número, e pode conter letra, "
+"números, hífens, pontos e sublinhados, com no máximo %[1]d characters.\n"
+"\t\t* Se --overwrite for true, então labels podem ser sobreescritos, caso "
+"contrário a sobreescrita irá falhar.\n"
+"\t\t* Se --resource-version for especificado, então as atualizações usarão "
+"esta versão do recurso, caso contrário, a versão do recurso existente será "
+"usada."
+
+#: pkg/kubectl/cmd/taint.go:58
+#, c-format
+msgid ""
+"\n"
+"\t\tUpdate the taints on one or more nodes.\n"
+"\n"
+"\t\t* A taint consists of a key, value, and effect. As an argument here, it "
+"is expressed as key=value:effect.\n"
+"\t\t* The key must begin with a letter or number, and may contain letters, "
+"numbers, hyphens, dots, and underscores, up to %[1]d characters.\n"
+"\t\t* The value must begin with a letter or number, and may contain letters, "
+"numbers, hyphens, dots, and underscores, up to %[2]d characters.\n"
+"\t\t* The effect must be NoSchedule, PreferNoSchedule or NoExecute.\n"
+"\t\t* Currently taint can only apply to node."
+msgstr ""
+"\n"
+"\t\tAtualiza os taints em um ou mais nodos.\n"
+"\n"
+"\t\t* Um taint consiste em uma chave, valor e efeito. Como arqgumento, é "
+"expressado como chave=valor:efeito.\n"
+"\t\t* Uma chave deve começar com uma letra ou número, e pode conter letra, "
```

Comment:
because the other words here are in plural (números, hívens) probably here should be "letras" instead of "letra"